src/styles/common.css

```diff
@@ -0,0 +1,338 @@
+/* Browser style Reset */
+@import url('./reset.css');
+/* Poppins font from google fonts */
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500&display=swap');
+/* Material Icons */
+@import url('https://fonts.googleapis.com/icon?family=Material+Icons');
+
+* {
+  box-sizing: border-box;
+}
+
+/* All css variables */
+:root {
+  font-family: 'Poppins', sans-serif;
+  --primary-color: #5851d8;
+  --primary-bg-color: #f2f7fd;
+  --secondary-bg-color: #ffffff;
+  --dark-text-color: #040405;
+  --primary-text-color: #0e1021;
+  --light-text-color: #31364e;
+  --fancy-dark-text-color: #3f2e53;
+  --light-grey-text: #c2b7b7;
```

Comment:
make variables for font size and spacing as well. For font size and spacing scale, checkout Refactoring UI, they've already included a pretty good scale that always worked for me.